app/controllers/authenticators_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+# REST endpoints for persisting authenticator configuration details in Conjur
+class AuthenticatorsController < RestController
+  include FindResource
+  include AuthorizeResource
+  
+  before_action :current_user
+  before_action :find_or_create_conjur_policy_branch
```

Comment:
Instead of using a before action, just call `find_or_create_conjur_policy_branch` instead of using `@conjur_policy_branch`.  Calling the method directly removes some indirection.